pages/integrations/inputs/o365_input.rst

```diff
@@ -0,0 +1,54 @@
+.. _o365_input:
+
+***********************
+Office 365 Integrations
+***********************
+
+.. attention:: This is a Graylog Enterprise feature and is only available since Graylog version 3.3. A valid Graylog Enterprise license is required.
+               It is meant for the hosted "online" office and do not use this to collect logs from a windows server, in your data center.
+
+Prerequisites
+=============
+You have an application in Azure AD, a subscription to Office 365 and a subscription to Azure.
+
+Why does this plugin exist ?
+============================
+Office 365 Integrations plugin, collects data from Windows Logs, Email, File Sharing and Data Loss Prevention Logs in one input.
+This plugin retrieves data for each tenant, using continuous polling and stores the data(user activity) in `elastic <https://en.wikipedia.org/wiki/Elasticsearch>`_ to detect threats.
+
+How do I get this plugin running ?
+==================================
+you should be able to login into the following websites with your exempli@gratia.onmicrosoft.com account.
+
+ - `office 365 <https://www.office.com/?auth=2>`_
+ - `azure Portal <https://portal.azure.com/#home>`_
+ - `turn on <https://protection.office.com/unifiedauditlog>`_ unified logging on friday and perform a search on monday,to see results.
```

Comment:
Did you want to tell that it need some time to be active?

Reply:
yes.